test/e2e/specs-playwright/wp-log-in-out-spec.js

```diff
@@ -1,32 +1,46 @@
 /**
  * External dependencies
  */
+import config from 'config';
 
-import playwright from 'playwright';
+/**
+ * Internal dependencies
+ */
+import * as browserManager from '../lib/browser-manager';
 
 import LoginPage from '../lib/pages/login-page';
 
+const mochaTimeOut = config.get( 'mochaTimeoutMS' );
+
 describe( `Auth Screen @canary @parallel`, function () {
-	this.timeout( 30000 );
+	this.timeout( mochaTimeOut );
+
+	// BrowserContext is equivalent to the `driver` used in Selenium.
+	let browserContext;
+	// Page represents a tab in a browser.
+	// Test steps interact with the page to execute its instructions.
 	let page;
-	let browser;
 
 	before( 'Start browser', async function () {
-		browser = await playwright.chromium.launch( {
-			headless: false,
-		} );
-		const browserContext = await browser.newContext();
+		browserContext = await browserManager.newBrowserContext();
+	} );
+
+	beforeEach( 'Open new test tab', async function () {
 		page = await browserContext.newPage();
+		// Set the page using mocha's metadata. Upon test failure,
+		// mocha hooks can access the page to perform actions.
+		this.currentTest.page = page;
 	} );
 
-	describe( 'Loading the log-in screen using Playwright', function () {
-		step( 'Can see the log in screen', async function () {
+	describe( 'Loading the log-in page', function () {
+		step( 'Can see the log in page', async function () {
 			const url = LoginPage.getLoginURL();
-			await page.goto( url, { waitUntill: 'networkidle' } );
+			/* 
+			Waits for network activity to cease.
+			Only as a proof of concept. In a production test, should check
+			for the presence of desired elements using a selector.
+			*/
+			return await page.goto( url, { waitUntill: 'networkidle' } );
 		} );
 	} );
-
-	after( 'close browser', function () {
```

Comment:
out of curiosity, why don't we need this now?

Reply:
In the next patch stack (which needs to be rebased on this branch after TS conversion), I connect the `after` hook to handle the browser closure. This way, the test writer doesn't need to be concerned with boilerplate code to handle browser closures.

Reply:
cool!